build.gradle

```diff
@@ -24,6 +24,12 @@ allprojects {
 	configurations.all {
 		resolutionStrategy.cacheChangingModulesFor 0, "minutes"
 	}
+
+	repositories {
+		maven {
```

Comment:
Looks like we can pull this now @cbornet 

https://repo.maven.apache.org/maven2/org/apache/pulsar/pulsar-client-reactive-api/0.2.0/

Reply:
Indeed. Done.